tests/test_agent_equipment_deployment_contract.py

```
@@ -64,6 +66,28 @@ def byte_digest(payload: bytes) -> str:
     return "sha256:" + hashlib.sha256(payload).hexdigest()
 
 
+def integer_constant_assignments(path: Path) -> dict[str, int]:
+    """Read simple integer limit assignments without importing candidate code."""
+
+    def evaluate(node: ast.expr) -> int:
+        if isinstance(node, ast.Constant) and type(node.value) is int:
+            return node.value
+        if isinstance(node, ast.BinOp) and isinstance(node.op, ast.Mult):
+            return evaluate(node.left) * evaluate(node.right)
+        raise ValueError("limit assignment must contain only integer multiplication")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)